src/ember-app.js

```diff
@@ -116,21 +118,34 @@ class EmberApp {
   buildWhitelistedRequire(whitelist, distPath) {
     whitelist.forEach(function(whitelistedModule) {
       debug("module whitelisted; module=%s", whitelistedModule);
+
+      // Remove after 2.0.0
```

Comment:
Curious to understand, why are we removing this after 2.0? This may be a backward incompatible change for apps that are relying on this.

Reply:
This is intended only for fastboot 1.x, with 2.0 all apps/addons should be using only package names in the whitelist, so it should be unnecessary here.

Do you think that schema version check should be used instead?